projects/plugins/jetpack/extensions/blocks/button/edit.js

```diff
@@ -45,20 +45,25 @@ export function ButtonEdit( props ) {
 
 	const blockClasses = classnames( 'wp-block-button', className );
 
-	const buttonClasses = classnames( 'wp-block-button__link', {
-		'has-background': backgroundColor.color || gradientValue,
-		[ backgroundColor.class ]: ! gradientValue && backgroundColor.class,
-		'has-text-color': textColor.color,
-		[ textColor.class ]: textColor.class,
-		[ gradientClass ]: gradientClass,
-		'no-border-radius': 0 === borderRadius,
-		'has-custom-width': !! width,
-	} );
+	const buttonClasses = classnames(
+		'wp-block-button__link',
+		{
+			'has-background': backgroundColor.color || gradientValue,
+			[ backgroundColor.class ]: ! gradientValue && backgroundColor.class,
+			'has-text-color': textColor.color,
+			[ textColor.class ]: textColor.class,
+			[ gradientClass ]: gradientClass,
+			'no-border-radius': 0 === borderRadius,
+			'has-custom-width': !! width,
+		},
+		fontSize && { [ `has-${ fontSize }-font-size` ]: true }
```

Comment:
I think this works too, so we can keep using just two arguments, right?

```suggestion
		{
			'has-background': backgroundColor.color || gradientValue,
			[ backgroundColor.class ]: ! gradientValue && backgroundColor.class,
			'has-text-color': textColor.color,
			[ textColor.class ]: textColor.class,
			[ gradientClass ]: gradientClass,
			'no-border-radius': 0 === borderRadius,
			'has-custom-width': !! width,
			[ `has-${ fontSize }-font-size` ]: !! fontSize,
		},
```

Reply:
Applied

Reply:
Looks like there some ESLint errors now, sorry about that!

Reply:
Fixed the linting errors.